src/test/java/step2/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,54 @@
+package step2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class StringAddCalculatorTest {
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    public void splitAndSum_null_또는_빈문자(String input) {
+        int result = StringAddCalculator.splitAndSum(input);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1"})
+    public void splitAndSum_숫자하나(String input) {
+        int result = StringAddCalculator.splitAndSum(input);
+        assertThat(result).isEqualTo(1);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1,2"})
+    public void splitAndSum_쉼표구분자(String input) {
+        int result = StringAddCalculator.splitAndSum(input);
+        assertThat(result).isEqualTo(3);
+    }
+
+    @Test
+    public void splitAndSum_쉼표_또는_콜론_구분자() {
+        int result = StringAddCalculator.splitAndSum("1,2:3");
+        assertThat(result).isEqualTo(6);
+    }
```

Comment:
```suggestion
    @ParameterizedTest(name = "입력: {0}, 결과: {1}")
    @CsvSource(value = {
            "1-1",
            "1,2-3",
            "1,2:3-6"
    }, delimiter = '-')
    public void splitAndSum(String input, int expected) {
        int result = StringAddCalculator.splitAndSum(input);
        assertThat(result).isEqualTo(expected);
    }
```
이렇게 여러개를 하나로 묶어 처리할 수 있을 것 같아요